Homework2/task1/src/homework/Main.java

```diff
@@ -0,0 +1,54 @@
+package homework;
+
+import homework.verification.*;
+
+/**
+ * Class Main has entry point to program.
+ * It has two methods: main and showInfo.
+ */
+public class Main {
+
+    /**
+     * Method showInfo gets boolean parameters and depending on values of the parameters
+     * output information, does the line complies to rules.
+     *
+     * @param noNumbers   boolean value: true - if there are no numbers in line.
+     * @param onlyNumbers boolean value: true - if there are only numbers in line.
+     * @param fiveWords   boolean value: true - if there are five or more words in line.
+     * @param dictWords   boolean value: true - if there are one or more words from dictionary in line.
+     */
+    public static void showInfo(boolean noNumbers, boolean onlyNumbers, boolean fiveWords, boolean dictWords) {
```

Comment:
What if you need to check 1000 rules? Will you add more method arguments? Put rule into array and iterate throught it